specreduce/tracing.py

```diff
@@ -346,6 +346,15 @@ def __post_init__(self):
                 # TODO: implement smoothing with provided width
                 y_bins[i] = ilum2[z_i.argmax()]
 
+        # warn about fully-masked bins (which, currently, means any bin with a single masked value)
+        if len(warn_bins) > 0:
+            plural = 's' if len(warn_bins) > 1 else ''
+            # if there are a ton of bins, we don't want to print them all out
+            if len(warn_bins) > 20:
+                warn_bins = warn_bins[0: 10] + ['...'] + [warn_bins[-1]]
+            warnings.warn(f"All pixels in bin{plural} {', '.join([str(x) for x in warn_bins])}"
+                          " are masked. Falling to trace value from all-bin fit.")
```

Comment:
I know this was the previous message, but maybe we should try to clarify while we're editing this - is this more accurate?  (If so, the test would need updating as well).

```suggestion
                          " are masked. Falling back on trace value from all-bin fit.")
```

Reply:
done